app/models/cdx/activity.rb

```diff
@@ -1,24 +1,35 @@
 class CDX::Activity < CDX::LoggedRequest
-  alias :create :perform
```

Comment:
yes I generally prefer to avoid aliased methods because they can be hard to grep for (would tend to grep for `def create` to find this method, which would not work)